tests/test_models/test_engine/test_db_storage.py

```diff
@@ -94,6 +94,40 @@ def test_get_obj_none(self):
         """check if obj is none"""
         self.assertEqual(None, models.storage.get("Tupac", 44))
 
+    @unittest.skipIf(models.storage_t != 'db', "testing db storage")
+    def test_no_args(self):
```

Comment:
In the section testing the method `get`, it might help readability to restate the method signature in the comments. I see you're passing zero and one arguments but not immediately clear why it raises a TypeError.